src/i965_drv_video.c

```diff
@@ -1378,6 +1431,10 @@ i965_suface_external_memory(VADriverContextP ctx,
         obj_surface->cb_cr_width = obj_surface->orig_width / 2;
         obj_surface->cb_cr_height = obj_surface->orig_height / 2;
         obj_surface->cb_cr_pitch = memory_attibute->pitches[1];
+        if (tiling)
+            ASSERT_RET(IS_ALIGNED(obj_surface->cb_cr_pitch,128),VA_STATUS_ERROR_INVALID_PARAMETER);
+        else
+            ASSERT_RET(IS_ALIGNED(obj_surface->cb_cr_pitch,i965->codec_info->min_linear_wpitch),VA_STATUS_ERROR_INVALID_PARAMETER);
 
```

Comment:
@xhaihao , the mentioned check is also used the IMC3 fourcc which supports the tiling format.
Even when the I420 fourcc is used,  it is harmless.  It has no side effect.